client/gutenberg/editor/hooks/components/media-upload/utils.js

```diff
@@ -47,3 +47,29 @@ export const mediaCalypsoToGutenberg = media => {
 		width: get( media, 'width' ),
 	};
 };
+
+export const getDisabledDataSources = allowedTypes => {
```

Comment:
Maybe it would be better to copy this code in some top level file in `client/gutenberg/` given that file like this one will be going away with Gutenlypso. This can also be done later at the time of that refactor too.